### PR TITLE
#lab 1

### PR DESCRIPTION
**Pull request by Raspy-Py**
Relese for lab-1 after fix.

### CODE REVIEW

**Review by Dichik — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es2021: true,
+  },
+  extends: [
```
Comment (nosovk) [inline .eslintrc.js:7]: sonar забыт

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lab_1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (nosovk) [inline package.json:7]: скрипты для lint/prettier etc?

File: src/Another-one.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css">
```
Comment (nosovk) [inline src/Another-one.html:6]: use absolute link, like /styles.css

File: src/Another-one.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        <li><a href="./Just-page.html">Just page</a></li>
+        <li><a href="./Another-one.html">Another one</a></li>
+      </ul>
+    </nav>
+    <section>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloribus, id ut inventore
+         saepe dolorem labore, ea ab in quam pariatur, dolores ipsa et consequuntur ipsam
+         consectetur quae facere suscipit fugit.</p>
+    </section>
+    <div class="left-content">
```
Comment (nosovk) [inline src/Another-one.html:29]: aside?

**Pull request closed without merging**